roles/users/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
   lineinfile: dest=/etc/sudoers
               state=present
               regexp="^%sudo"
-              line="%sudo ALL=(ALL:ALL) ALL"
+              line="%sudo ALL={{ sudoer_passwordless | ternary('(ALL) NOPASSWD:', '(ALL:ALL)') }} ALL"
```

Comment:
@chriszarate just realized an alternative solution I think might be better. Just have a variable called `sudo_group_permissions` which defaults to `(ALL:ALL)` and then you could customize it to `(ALL) NOPASSWD:`